mkfiles/stage1.mk

```diff
@@ -31,6 +31,7 @@ include $(ISO_SOURCE)/conf/modules.toolchain
 
 $(ISO_TARGET)/.stage1-toolchain: stage1-moonbase stage1-spool
 	@echo stage1-toolchain
+	@rm -f $(ISO_TARGET)/var/lib/lunar/plugins/postbuild-sanity.plugin
```

Comment:
I don't know a lot of details about the iso-building process.
Is this re-added in the final iso?